playbooks/provision-idm/README.md

```diff
@@ -0,0 +1,16 @@
+# IdM Provisioning playbook
+
+This playbooks runs through the steps to provision a VM, set it up, configure DNS records, and install IdM.
+Currently it is configured to provision OpenStack resources, but other providers can easily be added. 
+
+
+## Prerequisites
+You must have a working DNS server which will be able to access your IdM VM in order to setup forward and reverse DNS records. You will need the DNS zone key names, key secrets, and key algorithms. If you used the provision-dns-server playbook in [infra-ansible](https://github.com/redhat-cop/infra-ansible/tree/master/playbooks/provision-dns-server), there is a print_dns_key flag you can pass to have it print out these values.
```

Comment:
Suggestion: 
Change: `You must have a working DNS server which will be able to access your IdM VM ...` to something like `A working DNS server that accepts "nsupdate" connections for the IdM VMs forward and reverse DNS records to be added/updated to the existing DNS zones.`

Reply:
I'd also recommend not adding anything about the `print_dns_key` here as that can quickly change on the DNS role/playbook side (and hence rending this write-up out-of-date), but maybe rather update the docs on that side - i.e.: for the `provision-dns-server` playbook.